framework/assets/yii.validation.js

```diff
@@ -436,5 +436,32 @@ yii.validation = (function ($) {
         }
     }
 
+    function validateMimeType(mimeTypes, fileType)
+    {
+        for (var i=0; i<mimeTypes.length; i++) {
+            if (mimeTypes[i].indexOf('*') !== -1 && validateMimeTypeMask(mimeTypes[i], fileType)) {
+                return true;
+            }
+
+            if (mimeTypes[i] == fileType) {
```

Comment:
I think combining them could make worse code readability. I agree with second check is simpler, I'll replace then.